packages/bitwuzla-bin/bitwuzla-bin.0.0.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Bitwuzla SMT solver executable"
+description: """
+
+Standalone installer for the SMT solver Bitwuzla.
+
+Bitwuzla is a Satisfiability Modulo Theories (SMT) solver for the theories of fixed-size bit-vectors, arrays and uninterpreted functions and their combinations. Its name is derived from an Austrian dialect expression that can be translated as “someone who tinkers with bits”."""
+maintainer: ["Frédéric Recoules <frederic.recoules@cea.fr>"]
+authors: ["Frédéric Recoules"]
+license: "MIT"
+tags: ["SMT solver" "SMT-COMP 2020" "QF_AUFBVFP"]
+homepage: "https://bitwuzla.github.io"
+bug-reports: "https://github.com/bitwuzla/ocaml-bitwuzla/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "conf-git" {build}
+  "conf-cmake" {build}
+  "conf-gcc" {build}
+  "conf-g++" {build}
+  "conf-gmp" {build}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+  ]
+]
+dev-repo: "git+https://github.com/bitwuzla/ocaml-bitwuzla.git"
+x-commit-hash: "e1f77a4a4b12c9a8b624c6d9f11b0409f9b092fa"
+url {
+  src:
+    "https://github.com/bitwuzla/ocaml-bitwuzla/releases/download/0.0.0/bitwuzla-bin-0.0.0.tbz"
+  checksum: [
+    "sha256=d4226ee5d1972caae12ea7bfde7ffd20a04fa81abb7a85a7432fb71be34b6207"
+    "sha512=a94a466ef8b299c18cde51dd82e1be61f357de0e5efe617320ee7a888aeb8f7cd53d278f1e70e80b24c43003afe9595b8ad97c91c07a866616003e39642b35e8"
+  ]
+}
+available: [ os-family = "debian" | os-distribution = "alpine" ]
```

Comment:
This seems a bit excessive. Is there any chance you could find the reason why?

Reply:
Well, I will investigate, but not quite soon (yet the errors are on the build of vendor foreign libraries).

Since it is only the first release, I think it is better to release it for people for which it can work right now and release new patches/versions to relax the constraints.